CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 0.54.0 / 2021-01-26
+
+* [FEATURE] Support SNS Receiver in AlertmanagerConfig CR. #4468
+* [ENHANCEMENT] Specify SA token automounting on pod-level for operator and prometheus operand. #4514
+* [ENHANCEMENT] Support following redirects and Oauth2 in HTTP Client config in raw alertmanager config secret. #4499
+* [ENHANCEMENT] Add validations for relabel configs. #4429
```

Comment:
that could be considered a BUGFIX

Reply:
Makes sense, it would be consistent with the other validation changes.